cli/cmd/edges.go

```diff
@@ -118,8 +118,6 @@ func validateEdgesRequestInputs(targets []pb.Resource, options *edgesOptions) er
 			return fmt.Errorf("Resource type is not supported: %s", target.Type)
 		case "all":
 			return fmt.Errorf("Resource type is not supported: %s", target.Type)
-		default:
```

Comment:
This `switch` can be simplified to:
```
switch target.Type {
  case "authority", "service", "all":
    return fmt.Errorf("Resource type is not supported: %s", target.Type)
}
```
Or just an `if` statement.

Reply:
🙌 good point, thanks!